practice-two/src/css/main.css

```diff
@@ -34,6 +34,43 @@
   display: none;
 }
 
+.best-location-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
Not sure if we use flex-start here or not. I think the content should be central

Reply:
i will check it and reply in DM!!!